lib/chef-dk/skeletons/code_generator/templates/default/helpers.rb.erb

```diff
@@ -0,0 +1,28 @@
+#
+# Chef Documentation
+# https://docs.chef.io/libraries.html
+#
+
+#
+# This module name was auto-generated from the cookbook name. This name is a
+# single word that starts with a capital letter and then continues to use
+# camel-casing throughout the remainder of the name.
+#
+module <%= @cookbook_class_name %>
+  module <%= @helper_class_name %>
+    #
+    # Define the methods that you would like to assist the work you do in recipes,
+    # resources, or templates.
+    #
+    # def my_helper_method
+    #   # help method implementation
+    # end
+  end
+end
+
+#
+# The module you have defined above may be included into ALL OF YOUR RECIPES by
+# including the following line here or within your recipes.
+#
+# Chef::Recipe.include <%= @cookbook_class_name %>::<%= @helper_class_name %>
```

Comment:
I had noticed that something like the following (which is placed in the library file, in this case under the end for `module OperatingSystemHelpers`) seems to have been adopted as being the best practice. Are you suggesting something else? (I'm not sure of the syntax for what you are suggesting @coderanger )
```
Chef::Recipe.send(:include, OperatingSystemHelpers)
Chef::Resource.send(:include, OperatingSystemHelpers)
Chef::Provider.send(:include, OperatingSystemHelpers)
```